user-service/src/handlers/deleteUser.ts

```diff
@@ -0,0 +1,29 @@
+import * as AWS from 'aws-sdk';
+import { buildResponse } from '../utils';
+
+const db = new AWS.DynamoDB.DocumentClient()
+
+export const handler = async (event: any = {}) => {
+    const userId = event.pathParameters.userId;
+
+    if (!userId) {
+        return buildResponse(400, 'Error: you are missing the path parameter id')
+    }
+
+    const params = {
+        TableName: 'users',
+        Key: {
+            id: userId
+        }
+    }
+
+    try {
+        await db.delete(params).promise();
+        return buildResponse(200, 'Successfully deleted')
+
+    } catch (err: unknown) {
+        return buildResponse(500, {
+            err
+        })
+    }
```

Comment:
Instead of doing this (delete, create, add, or whatewer) better implement some service called `User` with the appropriate methods. And then use it wherever you need:
```
class User {
   static get(id): {
      // ... some code  
   }
   
   static create(user): {
      // ... some code  
   }

   static update(user): {
      // ... some code  
   }

   static delete(id): {
      // ... some code  
   }
}
```

And then usage:
```
const createHandler = () => {
     // ...some code
     await User.create(user)
}

const deleteHandler = () => {
     // ...some code
     await User.delete(user.id)
}
```

I hope you get the point